### ISSUE TITLE
I cannot access MISP dashboard

### ISSUE THREAD

**Issue by FarisThabit** [tguard-soc-package/nusantara#3]
I am in the process of installing T-Guard components. After following all the steps in the setup GUI interface (I will attach a picture)
![Wazzuh-Setup](https://github.com/tguard-soc-package/nusantara/assets/125390224/3ac07f53-6e17-401c-912f-6cc2abe61e86)
, I can access the dashboard of (Wazuh, IRIS, and Shuffle) using the credential information provided in T-Guard documentation (I will attach a picture of the credential information provided in T-Guard documentation).
![credential information provided in T-Guard documentation](https://github.com/tguard-soc-package/nusantara/assets/125390224/4cd78917-43ad-4eb6-a2c8-1d5c9f7dabb7)
 Still, I cannot access MISP dashboard although all the containers are running (I will attach a picture of all the running containers related to MISP).
![MISP containers are ready](https://github.com/tguard-soc-package/nusantara/assets/125390224/67e2d183-417a-4b6e-8c99-ad0c222e4433)
When I try to access MISP dashboard using (https://:1443), the page freezes.
![No Connection to MISP](https://github.com/tguard-soc-package/nusantara/assets/125390224/5b49bc0e-2efc-483a-a2b6-6dda2cf27b19)

Could you please help me to debug the issue?
I have been trying to solve the issue for a while, but no promising result.

**Comment by farukerdem34** [comment]
Have you check your firewall configs? Looks like you are on Ubuntu. 

To check ufw status
```bash
sudo ufw status
```
If it is active and port `1443` is not open, you have to use 
```bash
sudo ufw allow 1443
```
or
```bash
sudo ufw allow from <YOUR-IP-ADDRESS> to any port 1443
```

**Comment by bradsap54** [comment]
Hi, Faris!

I am currently involved in the T-Guard Development Team.

Thank you for your feedback on using our T-Guard SOC Package. Let’s discuss this issue in the T-Guard v1.0 Support category on the T-Guard Community website here: 
https://community.tguard.org/

Don’t worry, we have already prepared the answer to your problem. 
See you there!

**Comment by FarisThabit** [comment]
Thank you for taking the time to reply,

At the start my firewall was inactive. I enabled it and checked the status using: sudo ufw status, but no port was open. I allowed the port 1443 using: sudo ufw allow 1443, and retried to reaccess the MISP dashboard. Still, I can't.
![Check ufw ports](https://github.com/user-attachments/assets/e41a2a4f-4760-42e4-83ee-42f762a11e03)

**Comment by bradsap54** [comment]
Hi, Faris!

Have you found a solution to this issue, and is it resolved?

If so, please let us know your update.

Thank you!

**Comment by FarisThabit** [comment]
> Hi, Faris!
> 
> Have you found a solution to this issue, and is it resolved?
> 
> If so, please let us know your update.
> 
> Thank you!

Yes, the issue has been resolved. For anyone facing the same issue, see this link:
https://community.tguard.org/t/i-cannot-access-misp-dashboard/29/2

**Comment by bradsap54** [comment]
Thank you so much for the update! We’re delighted to hear that the issue has been resolved. 😊 

If there is any other questions or need further assistance, please feel free to reach out at our community support (https://community.tguard.org/).

Best regards, 
T-Guard Team
